fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LucenOnlineIndexingTest.java

```diff
@@ -502,8 +502,19 @@ private TestRecordsTextProto.MapDocument multiEntryMapDoc(long id, String text,
 
     private String[] listFiles(Index index) {
         try (FDBRecordContext context = openContext()) {
-            final FDBDirectory directory = new FDBDirectory(recordStore.indexSubspace(index), context, index.getOptions());
-            return directory.listAll();
+            if (index.getRootExpression() instanceof GroupingKeyExpression) {
+                List<String> files = new ArrayList<>();
+                for (int i = 0; i < 2; i++) {
```

Comment:
This won't necessarily catch all groups, right?
As a followup we should probably align this more with what we do in `OnlineIndexer.mergeIndex` where we gather all the groups and list them.
It might also make sense to return a `Map<Tuple, String[]>`, returning the results of each `listAll` associated with the group tuple.

Reply:
We should probably followup to make sure this assertion covers all groups.

Reply:
I did not make this change - this is part of the PR you merged yesterday, and is likely some PR issue that confused the diff.